### PR TITLE
Add entity to base event - cake 5.x

### PR DESCRIPTION
**Pull request by cnizzardini**
Ports this into master: https://github.com/lorenzo/audit-stash/pull/65

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -5,14 +5,12 @@ on:
...
+    name: PHP ${{ matrix.php-versions }} Test
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         operating-system: [ ubuntu-20.04 ]
-        php-versions: ['8.1']
-
-    name: PHP ${{ matrix.php-versions }} Test
+        php-versions: ['8.1', '8.2', '8.3']
```
Comment (dereuromark) [inline .github/workflows/pull-request.yml]:
```suggestion
        php-versions: ['8.1', '8.3']
```

Comment (cnizzardini) [reply .github/workflows/pull-request.yml]: Why is `8.2` not necessary? Is this considered overkill?

Comment (dereuromark) [reply .github/workflows/pull-request.yml]: Jep

**Comment by dereuromark** [comment]
Has this been merged in yet?

**Comment by cnizzardini** [comment]
> Has this been merged in yet?

No, only because I am reluctant to merge something I haven't had time to personally test. All my projects using this functionality are CakePHP 4.x. I guess its okay to be merged...

**Comment by dereuromark** [comment]
Maybe you can re-prepare it and we can see if things work out in CI.

**Comment by cnizzardini** [comment]
@dereuromark messed this up, just creatd a new PR/branch here: #81

**Pull request closed without merging**